src/transforms/route.rs

```diff
@@ -170,4 +169,15 @@ mod test {
         )
         .unwrap();
     }
+
+    #[test]
+    fn can_serialize() {
+        // We need to serialize the config to check if a config has
+        // changed when reloading.
+        let config = LaneConfig {
+            condition: AnyCondition::String("foo".to_string()),
+        };
+
+        assert!(serde_json::to_vec(&config).is_ok());
```

Comment:
This is going to be a fixed string, so I think this should compare with `assert_eq!(serde_json…, Ok(…));`